src/Renderer/AirspacePreviewRenderer.cpp

```diff
@@ -90,18 +90,18 @@ AirspacePreviewRenderer::UnprepareFill([[maybe_unused]] Canvas &canvas)
 
 bool
 AirspacePreviewRenderer::PrepareOutline(
-    Canvas &canvas, AirspaceClass type, const AirspaceLook &look,
+    Canvas &canvas, AirspaceClass asclass, const AirspaceLook &look,
     const AirspaceRendererSettings &settings)
 {
-  const AirspaceClassRendererSettings &class_settings = settings.classes[type];
+  const AirspaceClassRendererSettings &class_settings = settings.classes[asclass];
```

Comment:
Renaming this variable just adds noise to the patch diff and makes it hard to review